assets/js/settings/blocks/feature-flags.ts

```diff
@@ -23,19 +23,20 @@ export const registerExperimentalBlockType = (
 	}
 };
 
-/**
- * Registers a new feature plugin block provided a unique name and an object
- * defining its behavior. Once registered, the block is made available as an
- * option to any editor interface where blocks are implemented.
- */
-export const registerFeaturePluginBlockType = (
-	blockNameOrMetadata: string | BlockConfiguration,
-	settings: Record< string, unknown >
-): Block | undefined => {
-	if ( WC_BLOCKS_PHASE > 1 ) {
-		return registerBlockType( blockNameOrMetadata, settings );
-	}
-};
+// This code is being kept in case we add a feature-plugin-only block in future. It will be easy to reinstate.
+// /**
+//  * Registers a new feature plugin block provided a unique name and an object
+//  * defining its behavior. Once registered, the block is made available as an
+//  * option to any editor interface where blocks are implemented.
+//  */
+// export const registerFeaturePluginBlockType = (
+// 	blockNameOrMetadata: string | BlockConfiguration,
+// 	settings: Record< string, unknown >
+// ): Block | undefined => {
+// 	if ( WC_BLOCKS_PHASE > 1 ) {
+// 		return registerBlockType( blockNameOrMetadata, settings );
+// 	}
+// };
```

Comment:
Are you commenting this out because it's not used anymore?

Reply:
Yes, I chose to do this instead of deletion because it will be easier to reinstate this if required in future. Easier than finding the PR that removed it I mean.

Reply:
I’m on the camp of leaving it intact. It will get treeshaken id not used so it’s not hurting anyone, better than someone using it out of memory just so it doesn’t work for them because it’s commented out. X is undefined are hard to track sometimes.